tests/components/synology_dsm/test_config_flow.py

```diff
@@ -48,9 +49,10 @@ def mock_controller_service():
         "homeassistant.components.synology_dsm.config_flow.SynologyDSM"
     ) as service_mock:
         service_mock.return_value.login = Mock(return_value=True)
```

Comment:
Following https://github.com/home-assistant/core/pull/33729#discussion_r404267226

You can remove this line.

Reply:
Its going to return a MagicMock instead of True if I do that but it should still work